exercises/ex00-writing-tests/README.md

```diff
@@ -18,11 +18,11 @@ In ```ex00-writing-tests/flipper/src/tests/``` you will find the ```mock.rs``` f
 
 ## Some links
 * Awesome Rusty : https://github.com/rusty-crewmates/awesome-rusty
-* Pallet skeleton : https://docs.substrate.io/v3/runtime/frame/#skeleton-of-a-pallet
-* Informations about the tests with Substrate : https://docs.substrate.io/v3/runtime/testing/
+* Pallet skeleton : https://docs.substrate.io/tutorials/work-with-pallets/custom-pallet/
+* Informations about the tests with Substrate : https://docs.substrate.io/main-docs/test/
 * Some macros you could need : https://docs.rs/frame-support/2.0.0-rc4/frame_support/#macros
 * About Substrate's Origin : https://docs.substrate.io/v3/runtime/origins/
```

Comment:
It looks like this link is also broken.
We should maybe replace it with https://docs.substrate.io/main-docs/build/origins/

Reply:
Ok then. I will do just that. Thank you